New Release updates (#247)
tigrisdata/tigris-docs: docs/guides/nextjs/index.mdx

data-rich features, seamlessly implement search, and easily connect
serverless functions.

### Getting started

<Installation />

### Create Tigris Project

Once you have signup into the Tigris console, click on the `Create a new project` button to create a new project from your Tigris Cloud dashboard.

> ![create a new project](/img/screenshots/create-new-project.png)

Reviewer: ```suggestion
![The create a project dialog within the Tigris console web application](/img/screenshots/create-new-project.png)
```

> Enter `todo` as your project name and click on the `Create` button. Tigris will generate the Cloud credentials for your application with a command to create your application using the credentials.

Reviewer: ```suggestion
Enter `todo` as your project name and click on the `Create` button. Tigris will generate the Cloud credentials for your application. You will also be shown a command to create your application using the credentials.
```

> Enter `todo` as your project name and click on the `Create` button. Tigris will generate the Cloud credentials for your application with a command to create your application using the credentials.

Reviewer: I don't understand this sentence:

> Tigris will generate the Cloud credentials for your application with a command to create your application using the credentials.

Author: Changed "should" to "will"

> Tigris uses your project name as the database name of your project, saving the time of having to set one up.

Reviewer 2: Change this line to:

Every Tigris project comes with a pre-configured database.

### Scaffold your Next.js app

To scaffold your application, copy the `create-tigris-app` command generated in your Tigris console.

> ![create-tigirs-app command](/img/screenshots/create-tigris-app-command.png)

Reviewer: ```suggestion
![The create-tigirs-app terminal command displayed within a dialog within the Tigris web console](/img/screenshots/create-tigris-app-command.png)
```

> ![create-tigirs-app-command](/img/screenshots/create-tigris-app-command.png)

Reviewer: Can we add a alt text description?

> ![create-tigirs-app-command](/img/screenshots/create-tigris-app-command.png)

Reviewer: Can we add a better alt text description?

Author: Done

This will prompt you to pick the template for your project, select the `nextjs-api-routes` template, and click enter.

### Run

Run the app and see it automatically create your collections.

```shell
cd todo
npm run dev
```

:tada: Your full-stack web app is ready and accessible on `localhost:3000` in the browser.

> Now, if you go back to your Tigris console, you should see the collections created for the project.

Reviewer: I'd suggest we avoid words like "should".

We _should_ be confident that things will work. "Should" suggestions there's a scenario where the collections may not be created.

See [this section on avoiding vague and cautious language](https://github.com/ably/docs/blob/main/writing-style-guide.md#avoid-vague-and-cautious-language 
) from a style guide by @tbedford.

Author: Changed "should" to "will"

> ![Tigris collections](/img/screenshots/created-collections.png)

Reviewer: ```suggestion
![The collections for the project within the Tigris web console](/img/screenshots/created-collections.png)
```

> ![created_collections](/img/screenshots/created-collections.png)

Reviewer: Improve alt text

Author: Done

### Guides

> Here you will find a set of guides to help you quickly build a dynamic data-rich

Reviewer: ```suggestion
Here you will find a set of guides to help you build a dynamic data-rich
```

> Here you will find a set of guides to help you quickly build a dynamic data-rich

Reviewer: easily/quicky - do we need either of these words or can it be removed completely?
